rkt/images.go

```diff
@@ -31,6 +31,12 @@ import (
 	"strings"
 	"time"
 
+	"github.com/coreos/rkt/common/apps"
+	"github.com/coreos/rkt/pkg/keystore"
+	"github.com/coreos/rkt/rkt/config"
+	"github.com/coreos/rkt/stage0"
+	"github.com/coreos/rkt/store"
```

Comment:
It does not complain if they are in different paragraph. It only needs to be sorted within paragraphs.